doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -156,7 +156,8 @@ CONTAINER ID        IMAGE               COMMAND             CREATED
 As seen above, the default configuration distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
```

Comment:
Should the fix in this PR be mentioned in CNR?

Reply:
fixed in 2b78f9ec66c23ba035202ace15cdcb73ebd13679